src/zenml/orchestrators/utils.py

```diff
@@ -50,6 +56,52 @@ def create_tfx_pipeline(
     )
 
 
+def get_cache_status(
+    execution_info: data_types.ExecutionInfo,
+) -> bool:
+    """Returns the caching status of a step.
+
+    Args:
+        execution_info: The execution info of a `tfx` step.
+
+    Raises:
+        AttributeError: If the execution info is `None`.
+        KeyError: If no pipeline info is found in the `execution_info`.
+
+    Returns:
+        The caching status of a `tfx` step as a boolean value.
+    """
+    if execution_info is None:
+        logger.warn("No execution info found when checking for cache status.")
+        return
+
+    status = False
+    repository = Repository()
+    # TODO [HIGH]: Get the current running stack instead of just the active
```

Comment:
Is this TODO still relevant?

Reply:
It is still relevant. You didn't want to query the metadata store, but rather wanted to do something creative with the `execution_info` information. We said to leave it for now, but that in the future we might want to reconsider this approach.